modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -104,6 +104,78 @@ You need all the new channels fully synchronized before bootstrapping any Ubuntu
 endif::[]
 
 
+== Mirror Ubuntu ESM packages
+
+Canonical provides https://ubuntu.com/security/esm[Expanded Security Maintenance] (**ESM**) packages for https://ubuntu.com/pro[Ubuntu Pro] users and customers. 
+These packages offer longer maintenance (_10 to 12 years_) for several operating system components and selected applications.
+
+These repositories can also be synchronized within {productname} if you extract the required **GPG keys** and your personal **Bearer Token** from a system registered to Ubuntu Pro.
+
+
+=== Extract GPG keys and Bearer Token
+
+Register an Ubuntu host with Ubuntu Pro. 
+You will find your personal registration token in the https://ubuntu.com/pro/dashboard[Ubuntu Pro Dashbaord]. 
+An https://login.ubuntu.com/[Ubuntu One acconut] is required for this.
+
+----
+sudo apt-get install ubuntu-advantage-tools
+sudo pro attach <perosnal_token>
+----
+
+After registration, you will find the Bearer Token in the file `/etc/apt/auth.conf.d/90ubuntu-advantage`:
+
+----
+machine esm.ubuntu.com/apps/ubuntu/ login bearer password <token>  # ubuntu-pro-client
+machine esm.ubuntu.com/infra/ubuntu/ login bearer password <token>  # ubuntu-pro-client
+----
+
+[NOTE]
+====
+One dedicated bearer token is used per repository.
+====
+
+Configure the following repositories within {productname}:
+
+=== Configure Ubuntu ESM repositories
+
+Use the following URLs for creating the repositories:
+
+[[ubuntu-esm-repositories]]
+[cols="1,1", options="header"]
+.Ubuntu ESM repositories
+|===
+
+| URL     | Description
+| https://bearer:<token>@esm.ubuntu.com/infra/ubuntu/dists/<release>-infra-updates/main/binary-<arch>/ | Operating system functional updates
+| https://bearer:<token>@esm.ubuntu.com/infra/ubuntu/dists/<release>-infra-security/main/binary-<arch>/ | Operating system security updates
+| https://bearer:<token>@esm.ubuntu.com/apps/ubuntu/dists/<release>-apps-updates/main/binary-<arch>/ | Application functional updates
+| https://bearer:<token>@esm.ubuntu.com/apps/ubuntu/dists/<release>-apps-security/main/binary-<arch>/ | Application security updates
+|===
+
+Replace `<token>` with your personal Bearer Token. Also, `arch` and `release` must be replaced with one of the following values:
+
+[[ubuntu-esm-archs-releases]]
+[cols="1,1", options="header"]
+.Ubuntu ESM architectures and releases
+|===
+
+| Architectures | Releases
+| `amd64`, `arm64`, `armel`, `armhf`, `i386`, `powerpc`, `ppc64el`, `s390x` | `bionic`, `focal`, `jammy`, `noble`, `trusty`, `xenial`
+|===
+
+In order for {productname} to synchronize the repositories, the corresponding GPG keys (`ubuntu-advantage-esm-infra-trusty.gpg`, `ubuntu-advantage-esm-apps.gpg`) must be imported. These are located on a system registered with Ubuntu Pro under `/etc/apt/trusted.gpg.d`. Copy these files to the {productname} system and import them as follows:
+
+----
+sudo gpg --homedir /var/lib/spacewalk/gpgdir --import ubuntu-advantage-esm-*.gpg
```

Comment:
I guess on SUMA 5.0 and containerized Uyuni, you must use `mgradm` now:

```suggestion
mgradm gpg add /path/to/gpg.key
```

See https://github.com/uyuni-project/uyuni-docs/pull/2982